service/http/getJSONWebKeys_test.go

```diff
@@ -17,24 +17,37 @@
 package http_test
 
 import (
+	"context"
 	"net/http"
 	"testing"
+	"time"
 
 	serviceHttp "github.com/plgd-dev/client-application/service/http"
 	"github.com/plgd-dev/client-application/test"
 	httpgwTest "github.com/plgd-dev/hub/v2/http-gateway/test"
+	"github.com/plgd-dev/kit/v2/codec/json"
 	"github.com/stretchr/testify/require"
 )
 
-func TestClientApplicationServerGetWebConfiguration(t *testing.T) {
-	cfg := test.MakeConfig(t)
-	cfg.APIs.HTTP.TLS.ClientCertificateRequired = false
-	cfg.APIs.HTTP.UI.Enabled = true
-	shutDown := test.New(t, cfg)
-	defer shutDown()
+func TestClientApplicationServerGetJSONWebKeys(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*3600)
```

Comment:
1 hour timeout seems a bit long for a test

Reply:
fixed